_i18n/en.yml

```diff
@@ -127,12 +127,32 @@ hangouts:
     monero-site: Where the development of this website is coordinated.
 
 merchants:
-  intro1: Merchants of all kinds have come to value the financial privacy that Monero brings. Below is a list of the merchants that we know of that currently accept Monero for their goods and services. <b>These merchants are not vetted by the Getmonero team, see the disclaimer at the bottom of this page</b>. If a company no longer accepts Monero or you would like your business to be listed, please
-  intro2:  open a GitHub issue and let us know
-  intro3: (select the "Merchant" template and fill up all sections).
+  descr: On this page we list merchant directories, where you can spend your Moneroj and P2P, decentralized and centralized exchanges.
```

Comment:
```suggestion
  descr: On this page you will find stores and merchants that accept Monero as payment, as well as exchanges that trade Monero for other currencies.
```

Reply:
A less technical description that explains what exchanges are used for

Reply:
I want to point out that we post both centralized and decentralized exchanges. We can remove `P2P` if that's seen too technical.

Reply:
P2P is too technical, as well as the centralized x decentralized differentiation.

I don't think it's needed to list the different types of exchanges on the first sentence.

I think it's better to first give a simple explanation of what an exchange is/what it is used for. Maybe for us it seems obvious, but many new users don't know what an "exchange" is, and we don't want to throw 3 more unknown terms for these users on the first sentence.